src/resources/LibraryStyles.css

```diff
@@ -397,7 +398,6 @@ input {
 .SearchResultItemContainer .ItemIcon {
     padding: 2px 8px;
     width: 40px;
-    height: 40px;
```

Comment:
Oh yup it doesn't work on resizing. Another workaround is that we set the `min-width` of `.ItemIcon` to be 40px, so that it won't be pushed to left by `.ItemInfo`, and the percentage width of `.ItemInfo` can also be removed.